html-web-page/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <div>
+    <h1>Jelly of the Month Club</h1>
+  </div>
+  <div>
+    <h2>Make all your friends jelly of your stock jelly!</h2>
+  </div>
+  <div>
+    Here at JOTMC Inc., we are constantly striving to supply
+    our customers with the highest quality and most unique jellies
+    on a monthly basis. For the jelly lover in your life (or yourself!),
+    this is the gift that keeps on giving all year round. We offer three
+    tiers of monthly jelly joy, and each one offers their own perks. Here
+    is a breakdown of each, as well as your monthly price:
+  </div>
+  <br />
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th></th>
+          <th scope="col">Basic Plan</th>
+          <th scope="col">Deluxe Plan</th>
+          <th scope="col">Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free bonus jars of preserves bi-monthly</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <br />
+  <div>
+    Like what you see? Become a part of the JOTMC family today!
+  </div>
+  <br />
+  <div>
+    <form action="index.html">
+      <label>
+        <input type="text" name="name" size="20" maxlength="30" placeholder="Your Name" />
+      </label>
+      <select name="plans">
+        <option id="select" value="select" selected="selected">
+          <label for="select">Select Your Plan:</label>
```

Comment:
Since this is in a dropdown menu, the `label` doesn't add anything new, so it is unnecessary here, as well as for the other `option` elements.